docs: improve contributing.md (#9942)
renovatebot/renovate: .github/contributing.md

## Code

If you would like to fix a bug or implement a feature, please fork the repository and create a Pull Request.
To learn how to setup your local workstation correctly read [docs/development/local-development.md](../docs/development/local-development.md).
Also skim the [docs/development](../docs/development/) folder, it contains a lot of helpful information on things like adding a new package manager, how Renovate branches work, design decisions and more.

> Before you start any Pull Request, it's recommended that you create an issue to discuss first if you have any doubts about requirements or implementation.

Author: ```suggestion
Before you start any Pull Request, it's recommended that you open a [discussion](https://github.com/renovatebot/renovate/discussions) first if you have any doubts about requirements or implementation.
```
